cartoframes/data/observatory/catalog/entity.py

```diff
@@ -147,9 +150,21 @@ def _get_credentials(self, credentials=None):
 
         return _credentials
 
+    def _get_remote_full_table_name(self, user_project, user_dataset, public_project):
+        project, dataset, table = self.id.split('.')
```

Comment:
I see we do the split thing in several places to take the `project`, `dataset` and `table_name`. I'd wrap that into a utils method or something.

Reply:
It is a bit weird, because you must know the order when doing the split. Probably, these entities (at least Dataset, Geography, and Variable) should have `project`, `dataset` and `table_name` as properties to avoid doing that. I would tackle it in a different PR